package.json

```diff
@@ -68,7 +69,7 @@
       "@oclif/plugin-help"
     ],
     "aliases": {
-      "aliasme": "@oclif/plugin-test-esm-1"
+      "aliasme": "@oclif/plugin-version"
```

Comment:
What does the `aliases` oclif config do?

Reply:
allows you to alias a plugin. In this case, `plugins install aliasme` would install `@oclif/plugin-version`

It's only here for testing purposes though. It would need to be added to the root plugin in order to be used